Sprint-1/exercises/decimal.js

```diff
@@ -1,9 +1,21 @@
-const num = 56.5678;
-
-// You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
-
-// Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
-// Create a variable called decimalPart and assign to it an expression that evaluates to 0.5678 ( the decimal part of num )
-// Create a variable called roundedNum and assign to it an expression that evaluates to 57 ( num rounded to the nearest whole number )
-
-// Log your variables to the console to check your answers
+const num = 56.5678;
+
+// You should look up Math functions for this exercise https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math
+
+// Create a variable called wholeNumberPart and assign to it an expression that evaluates to 56 ( the whole number part of num )
+const wholeNumberPart=console.log(`the whole part of number${num} is : ${Math.trunc(num)}`);

```

Comment:
Here, you are declaring a variable called `wholeNumberPart` and assigning it the value returned by the call to the `console.log` function.
- What do you think the value of `wholeNumberPart` will be after this line? Can you output the value to find out?
- What happens if you remove this line? How does it affect the rest of the code? 🤔 
Can you try the above, running your code after each change?